client/styles/layout/commons/_commons.scss

```diff
@@ -12,7 +12,11 @@ $red: #d2333b;
     padding-bottom: 40px;
 
     @media(max-width: $screen-phone) {
-        padding: 0;
+        padding: 0 !important;
+
+        > form {
```

Comment:
Seems like you will have only one `form` in this context, so you don't need in `>` selector. If multiple forms expected, please add class to form you need and use this class as selector.